plugins/woocommerce-blocks/assets/js/base/components/sidebar-layout/style.scss

```diff
@@ -4,6 +4,10 @@
 	margin: 0 auto $gap;
 	position: relative;
 
+	&:has(> :last-child.wp-block-woocommerce-checkout-fields-block) {
```

Comment:
This CSS selector only works for the new checkout block markup. This prevents regressions on stores with the old markup.